components/Alert.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from '../styles/components/alert.module.scss';
+import Button from './ui/Button';
+
+const Alert = ({ message, visible, onClose }) => (
+  <div
+    className={visible ? [styles.popUpWrapper, styles.show].join(' ')
+      : styles.popUpWrapper}
```

Comment:
Component name is `Alert` while in styles you reference it as `Popup`. Please stick to a single naming.